### PR TITLE
Feature/jfdd14 umd 42 modal from email to game

### PR DESCRIPTION
**Pull request by SabinaSadowska**

### CODE REVIEW

**Review by zazulec — APPROVED** [review]
:) dla mnie spoko

File: game-modal.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    height: 60%;
+    background-color: var(--color-light-blue);
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    align-items: center;
+    justify-content: center;
+    border: 2px solid black;
+}
+
```
Comment (pasrobert) [inline game-modal.css:31]: delete space :)

**Review by pasrobert — APPROVED** [review]

File: scripts/form.js [hunk 2]
```diff
@@ -24,8 +25,8 @@ function emailToStorage(event) {
         alert("Niepoprawny adres email")
     } else {
         localStorage.setItem("email", inputValue);
-        alert("Dziękujemy :)")
-
+        console.log(modal)
```
Comment (JuliaLojek) [inline scripts/form.js:28]: usuń console.loga :D

File: scripts/game-modal.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const modalWIndow = document.querySelector('.game-modal');
+const btnQuit = document.querySelector('.game-modal__btn--quit');
+const btnPlay = document.querySelector('.game-modal__btn--play');
+
+function closeModal(){
+    modalWIndow.classList.remove('game-modal--active')
+}
+
```
Comment (JuliaLojek) [inline scripts/game-modal.js:8]: dużo enterów

**Review by JuliaLojek — APPROVED** [review]

**Review by pasrobert — APPROVED** [review]

**Pull request merged**